app/views/payments/index.html.erb

```diff
@@ -1,7 +1,7 @@
 <div id="payments" class="page-section">
   <% if @payments.any? %>
     <div class="d-flex justify-content-between">
-      <h2>History</h2>
+      <h1>History</h1>
```

Comment:
```suggestion
      <h1 class="h2">History</h1>
```

It looks like this causes  the styling to change on the payment page after we load the history dynamically:
![Screenshot 2024-03-20 at 09 52 50](https://github.com/sul-dlss/mylibrary/assets/111218/b6b59309-ff51-4cc7-a94e-1c31085905b0)

![Screenshot 2024-03-20 at 09 53 12](https://github.com/sul-dlss/mylibrary/assets/111218/757ca3e4-8a69-4c73-819b-be8c62774748)